typing-react-components/Exercise1/problem/App.tsx

```diff
@@ -0,0 +1,33 @@
+// 1. Add appropriate type to the Message component to get rid of TS errors.
+//    The type should not allow Message to be rendered without "children" i.e.
+//    the third Message render call in the code should be invalid.
```

Comment:
I think this line starts off with an error in CodeSandbox, so I wonder if there’s something else we can do for this or if it’s not worth including?

Reply:
This was actually added to show that we must always add a `children` with <Message> component and we need to type the `children` accordingly. So, if we use `PropsWithChildren` this error will go away but if we explicitly define `children` with `React.ReactNode`, the error shows that `children` must be defined. Basically, I want people to use `ReactNode` to type children instead of `PropsWithChildren`. Is there a better way to explain this?

Reply:
I have updated the problem statement to say "The type should require Message to be rendered with "children" i.e.
//    the third Message render call in the code should remain invalid." Let me know if you think I can explain this in a better way.